From a9f7a91f9fc53b7a766a5f081af82c9d79e89c1c Mon Sep 17 00:00:00 2001
Message-Id: <a9f7a91f9fc53b7a766a5f081af82c9d79e89c1c.1418787305.git.jen@redhat.com>
In-Reply-To: <c0f9fa126e925a00235d9077b474dabb4dbdfdc1.1418787305.git.jen@redhat.com>
References: <c0f9fa126e925a00235d9077b474dabb4dbdfdc1.1418787305.git.jen@redhat.com>
From: Jeffrey Cody <jcody@redhat.com>
Date: Wed, 3 Dec 2014 20:00:45 -0600
Subject: [CHANGE 08/12] block: remove BLOCK_OPT_NOCOW from vpc_create_opts
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Jeffrey Cody <jcody@redhat.com>
Message-id: <9b58fa0357d6830137fce214295396ffec437d13.1417620879.git.jcody@redhat.com>
Patchwork-id: 62667
O-Subject: [PATCH qemu-kvm-rhev RHEV7.1 v2 7/7] block: remove BLOCK_OPT_NOCOW from vpc_create_opts
Bugzilla: 1136381
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Max Reitz <mreitz@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: John Snow <jsnow@redhat.com>

In commit fef6070, the need for NOCOW was removed from the vpc driver,
as we removed the the posix calls.  However, the BLOCK_OPT_NOCOW was not
removed from vpc_create_opts.  This was a mistake - remove the opt from
there as well.

RHEL7 Note: Patch submitted upstream

Signed-off-by: Jeff Cody <jcody@redhat.com>
Signed-off-by: Jeff E. Nelson <jen@redhat.com>
---
 block/vpc.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/block/vpc.c b/block/vpc.c
index 9690344..3c794b5 100644
--- a/block/vpc.c
+++ b/block/vpc.c
@@ -888,11 +888,6 @@ static QemuOptsList vpc_create_opts = {
                 "Type of virtual hard disk format. Supported formats are "
                 "{dynamic (default) | fixed} "
         },
-        {
-            .name = BLOCK_OPT_NOCOW,
-            .type = QEMU_OPT_BOOL,
-            .help = "Turn off copy-on-write (valid only on btrfs)"
-        },
         { /* end of list */ }
     }
 };
-- 
2.1.0

